code/core/src/androidTest/java/com/adobe/marketing/mobile/services/ui/message/views/MessageScreenTests.kt

```diff
@@ -533,6 +533,131 @@ class MessageScreenTests {
         }
     }
 
+    @Test
+    fun testMessageScreenSizeWhenMaxWidthIsSetWithMaxHeightAndWidth() {
```

Comment:
Can you also add one test case where the maxWidth is not considered because it is larger than the specified `width`%